bash/cmd-line-searches/howmuch

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#================================================================================================================
+#title           :howmuch
+#description     :Presents a list of size of files in current directory human readably, and sorted descending
+#author          :dharvey-consbio
+#usage           :see https://github.com/consbio/cookbook/tree/master/bash/cmd-line-searches/#howmuch
+#=================================================================================================================
+
+du -chs .[!.]* * |sort -r --human-numeric-sort |head -11
```

Comment:
You can avoid the problem described above ("If there are no hidden files in the directory, you may get this message"), by testing for `.*` files first:

```bash
if [ -e ".*" ]; then
    du -chs .[!.]* * |sort -r --human-numeric-sort |head -11
else
    du -chs * |sort -r --human-numeric-sort |head -11
fi
```

Reply:
Very nice! I'll do that as well.

Reply:
It took a while to figure this out, but for that to work I needed:
```bash
export GLOBIGNORE=".:.."
```
Otherwise it always hit the second condition.

Reply:
Oh, interesting! I didn't have that behavior.